tools/gulptasks/dashboards/_config.json

```diff
@@ -11,5 +11,13 @@
     "namespace": "Dashboards",
     "product": "Highcharts Dashboards",
     "readmeFile": "ts/Dashboards/README.md",
-    "typeScriptFolder": "ts/masters-dashboards/"
+    "typeScriptFolder": "ts/masters-dashboards/",
+
+    "buildFolderDataGrid": "build/dist/datagrid/",
+    "bundleSourceFolderDataGrid": "code/datagrid/es-modules/masters-datagrid/",
```

Comment:
This should be `code/datagrid/es-modules/masters/` for consistency. Only in the `ts/` folder we have to distinguish more.